tls/s2n_security_policies.c

```diff
@@ -568,11 +568,11 @@ int s2n_security_policies_init()
             S2N_ERROR_IF(s2n_is_valid_tls13_cipher(cipher->iana_value) ^
                 (cipher->minimum_required_tls_version >= S2N_TLS13), S2N_ERR_INVALID_SECURITY_POLICY);
 
-            if (cipher->key_exchange_alg == &s2n_ecdhe || cipher->key_exchange_alg == &s2n_hybrid_ecdhe_kem) {
+            if (s2n_kex_includes(cipher->key_exchange_alg, &s2n_ecdhe)) {
```

Comment:
Why is the check for`s2n_hybrid_ecdhe_kem` removed? We require `ecc_extension_required` to be set to 1 for `cipher->key_exchange_alg == &s2n_hybrid_ecdhe_kem` as well.

Reply:
The logic here hasn't really changed.

I'm replacing the explicit check for s2n_hybrid_ecdhe_kem with what that check actually represented: any kex method that supports kem. If we added a new hybrid kex that includes s2n_kem, this will also be true for that new hybrid kex. 

If you look at the definition of ["s2n_kex_includes"](https://github.com/awslabs/s2n/blob/71f93064e2e23ab411c9f0cd7daf6524254ce0d1/tls/s2n_kex.c#L238-L248), it is true if a) the kex method is the query 2) the kex method is a hybrid that includes the query